src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +213,28 @@ private void validateSchema(final EventTypeBase eventType) throws InvalidEventTy
         }
     }
 
+    /**
+     * bugifx ARUHA-563
+     *
+     * @param eventTypeSchema
+     * @throws InvalidEventTypeException
+     */
+    private void checkSchemaValidity(final String eventTypeSchema) throws InvalidEventTypeException {
+        final char[] chars = eventTypeSchema.toCharArray();
+        int bracketsCounter = 0;
+        for (int i = 0; i < chars.length; i++) {
+            if (chars[i] == '{' || chars[i] == '[') {
+                bracketsCounter++;
+            } else if (chars[i] == '}' || chars[i] == ']') {
+                bracketsCounter--;
+            }
+            // if all brackets were closed but we still have symbols in schema then something wrong with the schema
```

Comment:
@SergKam it will look like:
`"{"additionalProperties": true}        "`
@lmontrieux 
```
"{"additionalProperties": true}

"
```

I do not think we should allow this.